.github/mergify.yml

```
@@ -550,3 +550,11 @@ merge_protections:
     success_conditions:
       - check-success = test/terratest
 
+queue_rules:
+  - batch_size: 10
+    batch_max_wait_time: 5 m
+    checks_timeout: 5 m
+    batch_max_failure_resolution_attempts: 3
+    merge_method: squash
+    update_method: rebase
+    name: Test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)